lambdas/helm-helper-lambda/helmHelperLambda.py

```diff
@@ -40,6 +41,16 @@ def handler(event, context):
             helmupgrade_doc = describe_document(event['ResourceProperties']['HelmUpgradeRunScript'])
             helmelb_doc = describe_document(event['ResourceProperties']['GetElbEndpointRunScript'])
 
+            # First download all helper scripts
+            init_doc = describe_document(event['ResourceProperties']['HelmDownloadScript'])
+            logger.info('Downloading helper scripts...')
+            init = ssm_sendcommand(ssm_instance, init_doc['Name'], {})
+            if ssm_commandstatus(init['CommandId'], ssm_instance) is True:
+                logger.info('scripts directory was downloaded successfully!')
+            else:
+                logger.error('Helper scripts download was unsuccessful')
+                cfnresponse.send(event, context, cfnresponse.FAILED, {})
```

Comment:
You're also missing a return statement here, if you're signalling failure to CFN we should not continuing executing, see REPO-4161 for a similar issue.